build.sc

```diff
@@ -96,10 +97,16 @@ object server extends ScalaModule {
     millSourcePath / "src" / "main" / "resources" / "localkey.pkcs12"
   }
 
+
+  override def compile = T {
```

Comment:
Downloading the fonts and compiling the server are two unrelated tasks (a failed download shouldn't fail compilation, for example). I think it makes sense to move this to `bundle` within the `Webpack` trait:


```scala
def bundle = T {
  vendor().map { p => os.copy.into(p.path, T.ctx.dest) }
  os.copy.into(sass().path, T.ctx.dest)
  os.copy.into(fonts().path, T.ctx.dest)
  ...
  PathRef(T.ctx.dest)
}
```